py/desisurvey/test/test_nextfield.py

```diff
@@ -51,9 +51,17 @@ def test_previoustiles(self):
         previoustiles = []
         for test in range(10):
             next_field = get_next_field(self.dateobs, self.skylevel, \
-                self.transparency, previoustiles, self.programname)
+                self.seeing, self.transparency, previoustiles, self.programname)
             self.assertNotIn(next_field['tileid'], previoustiles)
             previoustiles.append(next_field['tileid'])
+            
+    def test_rightanswer(self):
+        """
+        Test that the tileid returned is correct for the specified date
+        """
+        next_field = get_next_field(self.dateobs, self.skylevel, self.seeing, \
+            self.transparency, self.previoustiles, self.programname)
+        self.assertEqual(next_field['tileid'], 23492)
```

Comment:
Since this right answer is very closely tied to the dateobs, it would be better to fix the dateobs directly in this test rather than using self.dateobs -- someone else may change the original definition of self.dateobs without realizing that it would break this test.  Also add a comment that this "right" answer was determined by hand so it doesn't look quite like such a mystery number.  Would be best to include a few different dateobs with a few different right answers to provide more checks.  e.g. calculate example dateobs when RA=0, 30, 60... is the meridian and look up the lowest dec tiles at those RAs.